src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx

```diff
@@ -480,3 +480,64 @@ export const buildOnClickHandler = (currentPlanId: string) => {
   }
   return clickHandler;
 };
+
+/**
+ * Map component popup handler
+ * @param map - mapbox map instance
+ * @param event - map event object
+ */
+
+export const buildMouseMoveHandler = () => {
+  function mouseHoverHandler(map: Map, event: EventData) {
+    const popup = new mapboxgl.Popup({
+      closeButton: false,
+    });
+    // Added this part to remove duplicate popup containers showing on map when hovering
+    const nodes: any = document.getElementsByClassName('mapboxgl-popup');
+    let node: any;
+    if (nodes && nodes.length) {
+      for (node of nodes) {
+        if (node.style) {
+          node.style.display = 'none';
+        }
+      }
+    }
+    let content: string = '';
+    let feature: Feature | any = {};
+    popup.remove();
+    // grab underlying features from map
+    const features: any = event.target.queryRenderedFeatures(event.point) as Dictionary[];
+    //
+    if (!features.length) {
+      return;
+    }
+    // loop through features, setting the name property to content var
+    for (feature of features) {
+      if (feature && feature.properties && feature.properties.name) {
+        content += `<div><center>${feature.properties.name}</center></div>`;
```

Comment:
Let's give this div a class name so that in the future if we want to style it we can do so easily.